.gitattributes.zos

```diff
@@ -1,5 +1,5 @@
 #################################################################################
-# Copyright (c) 2017 IBM Corp.
+# Copyright (c) 2017, 2021 IBM Corp. and others
```

Comment:
Please remove the "and others" from this and the rest of the changes.

Reply:
Should I restore it to ` Copyright (c) 2017 IBM Corp. `? Also when I changed the end dates of the copyright of the following files to 2021, there were copyright errors so I have restored the copyright to what it was earlier.

Reply:
As far as checks of this PR is concerned - you only need to update the copyright of the files you had touched. You don't need to update every single file of the entire repo.

Reply:
About my comment above, what I meant is : 

The copyright should read like this: "Copyright (c) 2017, 2021 IBM Corp." 

Not this: "Copyright (c) 2017, 2021 IBM Corp. and others" 

It appears you had added two extra words by mistake in your change : "and others". These two words should be removed.

Reply:
> Pls update the IBM copyright end date in the files that contain IBM copyrights.
> The scripts that contain `Copyright (c) 2017 IBM Corp.` should be updated to `Copyright (c) 2017, 2021 IBM Corp. and others`
> 
> ```
> 23:32:17  The following files were modified and have incorrect copyrights
> [Pipeline] echo
> 23:32:17  openj9.build/docs/build.md
> [Pipeline] echo
> 23:32:17  openj9.build/scripts/openj9-systemtest-clone-make.bat
> [Pipeline] echo
> 23:32:17  openj9.build/scripts/openj9-systemtest-clone-make.sh
> [Pipeline] echo
> 23:32:17  openj9.stf.extensions/src/stf.extensions/net/openj9/stf/sharedClasses/StfSharedClassesExtension.java
> [Pipeline] echo
> 23:32:17  openj9.test.sharedClasses.jvmti/src/test.sharedClasses.jvmti/net/openj9/sc/api/SharedClassesCacheChecker.java
> ```

I thought that I had to update the copyrights of the all the files that contain the copyright.

Reply:
@Mesbah-Alam if you look at my previous comments, I specifically asked that "and others" be added. It should be there.

Reply:
Technically, you can update the copyright of every single file of the repo, as the copyright in every single file does need to be updated at some point. However, the automated check here only validates the copyright in the files you had change to implement the PR. So, if you only update copyright in those subset of files, that'd be enough, as far as merging this PR is concerned.

Reply:
The copyright end year should only be updated in files that are changed. The end year should match with the last update to the file.

Reply:
>@Mesbah-Alam if you look at my previous comments, I specifically asked that "and others" be added. It should be there. 

Oh... my bad! I hadn't notice that. 

So, to be clear: The content of the change is perfect - what we need is for it to appear in only the files changed in this PR, not the entire repo.

Reply:
I have reverted the previous commits and now updated the files changed in this PR only. Sorry for misunderstanding the task and thank you for helping me!

Reply:
You have one more README.md file to update. Please check the "FAILURE" here: https://ci.eclipse.org/openj9/job/PullRequest-CopyrightCheck-Systemtest/171/console 

You can copy the copyright header from openj9.build/docs/build.md and paste it in openj9.test.sharedClasses/docs/README.md

Reply:
Done

Reply:
That's the wrong READM.md file, we want the one at the top level which you have otherwise modified to fix the URLs.

Reply:
Okay, I will add the header to the top-level README.md and should I remove the header from openj9.test.sharedClasses/docs/README.md?

Reply:
You can leave it. You didn't need to add it as part of this change, but since you've already added it we might as well keep it.

Reply:
The copyright check still has failures: https://ci.eclipse.org/openj9/job/PullRequest-CopyrightCheck-Systemtest/172/console

Reply:
lgtm. Can you squash the commits?

Reply:
Done